data/hai missions.txt

```diff
@@ -1061,3 +1061,254 @@ mission "Hai Honeymoon"
+			`	You walk over to the group to help her, and get a better look at the family: a Hai couple carrying a baby, and three older children who aren't doing much to help their father carry their ludicrous number of bags.`
+			choice
+				`	"Hey, what's going on?"`
+				`	"Can I be of assistance?"`
+			`	You introduce yourself and the parents tell you that they've been on vacation here and need transport back to Hai-home. "Don't take this the wrong way, but captain Nanachi told us her ship is one of your human ships, a Heavy Shuttle was it? It has enough bunks for us but our sons already caused a ruckus on the way here in a Hai ship, we're worried that since they aren't used to the accommodations, they might end up giving her too much trouble."`
+			`	You reassure them that the trip will be very quick, and that the ship's interior is somewhat similar to that of Hai ships, until they finally agree on going with her. They settle on the pay and you two help the Hai get their baggage to Nanachi's ship. When the family is focused on a conversation, you quietly ask Nanachi what was the issue she was having with the other people she encountered.`
+			`	"They all seemed willing to give me a job, but I ended up blowing it when I asked for more money. I just really need to get out of debt so I guess I went a bit overboard." She tries to play it cool, but it's clear that she is embarrassed about losing so many potential jobs. When you reach her ship, you take a look at the prices for commodities in Hai-home. You decide to tell Nanachi to stock up on heavy metals to sell there, to help her make a bit more profit.`
+				accept
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for her to arrive in this star system.`
+	on complete
+		conversation
+			`You help the family gather all their luggage and bid them farewell, but right as they disappear into a hallway Nanachi lets out a heavy sigh. "I bit off more than I could chew. Those kids kept wanting to play with the ship's controls, and one of them tried to go up the air vent in a game of hide-and-seek. I couldn't get any rest since I had to keep making sure they weren't about to dump their own bags into the void by accident."`
+			`	"Do you want to get some sleep before trying to find another mission?" you ask, hoping she won't try to overwork herself. "Yes, I could really use that. Sorry to make you wait again captain <last>. I'll rest a bit, see if I can't find a tourist spot nearby to relax, and meet up with you the in spaceport after a while."`
```

Comment:
The Hai got to exciting places as tourists, so I'm not sure you want a tourist spot to "relax". (Although any tourist spot is more likely to be busy then somewhere else.)
Perhaps a park is more in order.

Reply:
It could also mean something like a touristy restaurant, or other area you can just go mindless in.  Personally, I see this like going to some chain restaurant, ordering some comfort food, and just "hiding in plain sight"/people-watch.  Considering she's also looking for work, a touristy spot might give a job offer...

Reply:
Changed to be a generic spot to relax.

Reply:
Changed with current version.